lms/djangoapps/discussion_api/serializers.py

```diff
@@ -187,6 +187,7 @@ class ThreadSerializer(_ContentSerializer):
     non_endorsed_comment_list_url = serializers.SerializerMethodField("get_non_endorsed_comment_list_url")
     read = serializers.BooleanField(read_only=True)
     has_endorsed = serializers.BooleanField(read_only=True, source="endorsed")
+    response_count = serializers.IntegerField(source="resp_total", read_only=True)
```

Comment:
required=False will omit field from output response but that is part of django rest-framework v3+ and we are using v2.3.14
for reference: http://www.django-rest-framework.org/topics/release-notes/